src/test/resources/db/migration/V1_1__init_tables.sql

```diff
@@ -0,0 +1,142 @@
+-- NB Flyway requires lowercase for table names
```

Comment:
These files have been added to **src/test/resources** (they are the same as those used in **src/main/resources**). This has been done so the database that is used in unit tests are built off these scripts and not those in **src/main** - as per discussions with @shashank-rastogi

Reply:
did you follow same in UP and Sync ? if not how did it work there ?

Reply:
Followed exact same in Sync, however in PRD we create two partial indexes which is supported in postgres but not yet supported by H2 so we cannot migrate our scripts via flyway due to this syntax deficiency in H2.

Reply:
Thats great finding 👍 . This justification will be helpful if some from QA ask us in future.

Reply:
I will add this to the **RDCC-1058 JIRA ticket** for future reference.

Reply:
remove all the scripts and create application.yml file inside test folder under  resources same as integration.
test/resources/application.yml

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/480/commits/dc4dc69e0df737dcc62fce07335d67d43f17c804